pkgs/development/compilers/go/bootstrap.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, srcOnly, fetchurl, callPackage, Security }:
+
+let
+go_bootstrap = if stdenv.isAarch64 then
+  srcOnly {
+    name = "go-1.8-linux-arm64-bootstrap";
+    src = fetchurl {
+      url = "https://cache.xor.us/go-1.8-linux-arm64-bootstrap.tar.xz";
```

Comment:
This file is still not accessible if not for cache.nixos.org:

```
* SSL: no alternative certificate subject name matches target host name 'cache.xor.us'
```

It also doesn't feel very trustworthy.

How do you feel about bootstrapping from golang.org binary packages repackaged to look like bootstrap? I have the following for aarch64-darwin locally:

```nix
      (srcOnly {
        name = "go-1.16-rc1-darwin-arm64-bootstrap";
        src = fetchurl {
          url = "https://golang.org/dl/go1.16rc1.darwin-arm64.tar.gz";
          sha256 = "1p9l9qrh5x06jw19bnyfz8hfq6fisxzhc39cq6gwg4xcvn7gxq7m";
        };
      }).overrideAttrs (_: rec {
        installPhase = ''
          mkdir -p $out/share/go $out/bin
          mv bin/* $out/bin
          cp -r . $out/share/go
        '';
      })
```

Same idea would apply to aarch64-linux here, or to any arch that cannot be bootstrapped from go1.4.

Reply:
This was partially discussed in #110794, IMO please open a new issue so we can discuss it. Personally, I think it would be fine.